modules/sandboxed-containers-enable-debug-crio.adoc

```diff
@@ -6,68 +6,36 @@
 [id="sandboxed-containers-enable-debug-logs_{context}"]
 = Enabling debug logs for {sandboxed-containers-first}
 
-As a cluster administrator, you can collect a more detailed level of logs for {sandboxed-containers-first}. Enhance logging by changing the `log_level` in the CRI-O runtime for the worker nodes running {sandboxed-containers-first}.
+As a cluster administrator, you can collect a more detailed level of logs for {sandboxed-containers-first}. Enhance logging by changing the `logLevel` field in the KataConfig CR. This changes the `log_level` in the CRI-O runtime for the worker nodes running {sandboxed-containers-first}.
 
 .Procedure
 
-. Create a YAML file for the `ContainerRuntimeConfig` CR with the following manifest:
-+
-[source,yaml]
-----
-apiVersion: machineconfiguration.openshift.io/v1
-kind: ContainerRuntimeConfig
-metadata:
- name: crio-debug
-spec:
- machineConfigPoolSelector:
-  matchLabels:
-    pools.operator.machineconfiguration.openshift.io/worker: '' <1>
- containerRuntimeConfig:
-    logLevel: debug
-----
-<1> Specify a label for the machine config pool that you want you want to modify.
+. Change the `logLevel` field in your existing KataConfig CR to `debug`:
 
-. Create the `ContainerRuntimeConfig` CR:
-+
 [source,terminal]
 ----
-$ oc create -f ctrcfg.yaml
+$ oc patch kataconfig cluster-kataconfig --type merge --patch '{"spec":{"logLevel":"debug"}}'
 ----
 
-+
 [NOTE]
 ====
-The file name listed above is a suggestion. You can create this file using another name.
+The object name for the KataConfig CR (`cluster-kataconfig`) listed above is the suggested name when creating your KataConfig CR. You can create this object using another name, however, make sure you reference the correct object name when running the above command.
 ====
 
-. Verify the CR is created:
-+
-[source,terminal]
-----
-$ oc get ctrcfg
-----
-+
-.Example output
-[source,terminal]
-----
-NAME           AGE
-crio-debug   3m19s
-----
-
 .Verification
 
-. Monitor the machine config pool until the `UPDATED` field for all worker nodes appears as `True`:
+. Monitor the `kata-oc` machine config pool until the `UPDATED` field for all worker nodes appears as `True`:
```

Comment:
Mentioning `kata-oc` is definitely a good change but the original text is a bit confusing. The `UPDATED` field tells whether **all** machine in the pool are updated (True) or at least **one** is still not up to date (False). I'd suggest :
```
 Monitor the `kata-oc` machine config pool until the `UPDATED` field appears as `True` which means all worker nodes are up-to-date.
```